robotbase/src/main/java/frc/robot/subsystems/PhotonVisionCamera.java

```diff
@@ -69,20 +63,41 @@ public void configure() {
    * outside of it.
    */
   public void updateResult() {
-    m_result = m_camera.getLatestResult();
-    m_targets = m_result.getTargets();
-    m_bestTarget = m_result.getBestTarget();
+    if(m_camera.isConnected()){
```

Comment:
I would prefer to minimize this check if it takes any amount of time. Is there a way we can check the return value of the update we do in Robot periodic to see if the camera is in a valid state?

Reply:
maybe? but I don't see the point in that, the isConnected method from the camera should be faster than getting the result. If we want to minimize the amount of times we call that, we could try making a system that manages multiple cameras. It shouldn't be very hard to do, if all it does is manage the connections and update the results.

Reply:
we also dont know if this is a real issue yet, the only time I've seen the cameras disconnect was when the power pack was less than half charged.